src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -83,6 +102,9 @@ public Map<String, String> getProperties(VariableResolver<String> resolver) {
                 if (m_sourceCodeOnly && getHasOptions()) {
                     properties.put(CoreConstants.SOURCE_CODE_ONLY, "");
                 }
+                if(m_scanSpeed!=null && !m_scanSpeed.isEmpty() && getHasOptions()) {
```

Comment:
Same as above regarding null vs. the empty string.

Reply:
Matt, It is throwing "null pointer exception" error...while doing the upgrade testing means executing the job which is created with the "1.1.0" version of the Jenkins plugin. 
![image](https://github.com/jenkinsci/appscan-plugin/assets/102226698/af18c187-fc98-494a-8e62-a34cebeb6ed0)